Sources/SKSupport/LineTable.swift

```diff
@@ -172,15 +172,37 @@ extension LineTable {
   /// - parameter utf8Column: UTF-8 column offset (zero-based).
   @inlinable
   public func utf16ColumnAt(line: Int, utf8Column: Int) -> Int? {
+    return convertColumn(
+      line: line,
+      column: utf8Column,
+      indexFunction: content.utf8.index,
+      distanceFunction: content.utf16.distance)
+  }
+
+  /// Returns UTF8 column offset at UTF16 version of logical position.
+  ///
+  /// - parameter line: Line number (zero-based).
+  /// - parameter utf16Column: UTF-16 column offset (zero-based).
+  @inlinable
+  public func utf8ColumnAt(line: Int, utf16Column: Int) -> Int? {
+    return convertColumn(
+      line: line,
+      column: utf16Column,
+      indexFunction: content.utf16.index,
```

Comment:
Please use the more-specific syntax `content.utf16.index(_:offsetBy:)`.  Same for `distance(from:to:)`.